src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The command class encapsulates an executable command by Duke.
+ */
+public abstract class Command {
+    /**
+     * The abstract method to execute the correct response to the user's command.
```

Comment:
To better follow the coding standards, maybe you could add an empty line between the description and parameter section?